src/AbstractFixer.php

```diff
@@ -63,6 +63,10 @@ public function __construct()
 
     final public function fix(\SplFileInfo $file, Tokens $tokens)
     {
+        if ($this instanceof ConfigurableFixerInterface && null === $this->configuration) {
```

Comment:
guess not https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/2865#discussion_r122938913
that was the point that started this, 
```
but if one won't pass any configuration (psr0 => true), then configs are not resolved and everything works properly
```